tcms/bugs/tests/test_permissions.py

```diff
@@ -58,6 +58,49 @@ def verify_post_with_permission(self):
         self.assertContains(response, 'BUG-%d' % last_bug.pk)
 
 
+class TestEdit(tests.PermissionsTestCase):
+    permission_label = 'bugs.change_bug'
+    http_method_names = ['get', 'post']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.bug = BugFactory()
+
+        cls.url = reverse('bugs-edit', args=(cls.bug.pk,))
+        cls.post_data = {
+            'summary': 'An edited summary',
+            'reporter': cls.bug.reporter.pk,
+            'assignee': cls.bug.assignee.pk,
+            'product': cls.bug.product.pk,
+            'version': cls.bug.version.pk,
+            'build': cls.bug.build.pk,
+        }
+
+        super().setUpTestData()
+
+    def verify_get_with_permission(self):
+        response = self.client.get(self.url)
+
+        self.assertContains(response, _('Edit bug'))
+        self.assertContains(response, self.url)
+        self.assertContains(response, self.bug.summary)
+        self.assertTemplateUsed(response, 'bugs/mutable.html')
+
+    def verify_post_with_permission(self):
+        response = self.client.post(self.url, self.post_data, follow=True)
```

Comment:
In order to view the page displaying bugs the user now needs bugs.view_bug permission as well. Assign it with user_should_have_perm. Can do it here, or in setup, doesn't matter.  Rebase to master to pick up the changes.